README.md

```diff
@@ -134,3 +134,17 @@ img, label = dataset[0]
 
 As found in the example, `random_rotate` is one of the transforms provided by ChainerCV. Like other transforms, this is just a
 function that takes an array as input.
+
+
+# Citation
+
+Please cite the paper if you would use it in your research:
```

Comment:
If ChainerCV helps your research, please cite the paper for ACM Multimedia Open Source competition.
Here is a BibTex entry: